### PR TITLE
currency list

### PR DESCRIPTION
**Pull request by jonnyalexbh**
## Summary
- coin listing

## Trello Card
N/A

## Screenshot
Screen Shot 2019-12-10 at 3 22 39 PM

### CODE REVIEW

File: plink-frontend/src/app/screens/currency-list/currency-list.component.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <div>Criptomoneda</div>
+  <div>Convertir</div>
+</div>
+
+<div class="currency-list" *ngFor="let price of prices">
+  <div>{{ price.name }}</div>
+  <div>{{ price.price }}</div>
+  <div>{{ price.crypto }}</div>
+  <div><img class="image-shuffle" src="../../../assets/shuffle.png" alt=""></div>
+</div>
```
Comment (betoobernal) [inline plink-frontend/src/app/screens/currency-list/currency-list.component.html]: use `span` tag when content will be only text

Comment (jonnyalexbh) [reply plink-frontend/src/app/screens/currency-list/currency-list.component.html]: ready 👍

File: plink-frontend/src/app/screens/currency-list/currency-list.component.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<div class="currency-header">
+  <div>Nombre</div>
+  <div>Precio</div>
+  <div>Criptomoneda</div>
+  <div>Convertir</div>
```
Comment (betoobernal) [inline plink-frontend/src/app/screens/currency-list/currency-list.component.html:5]: use span tag

Comment (jonnyalexbh) [reply plink-frontend/src/app/screens/currency-list/currency-list.component.html:5]: ready 👍

**Review by betoobernal — APPROVED** [review]

**Pull request merged**